qprofiler2/src/org/qcmg/qprofiler2/Options2.java

```diff
@@ -67,43 +67,46 @@
 		parser.posixlyCorrect(true);
 		options = parser.parse(args);
 		
-		// no of threads - Consumer
+		 //  no of threads - Consumer
 		Object threadNumberConsumer = options.valueOf("ntConsumer"); 
-		if (null != threadNumberConsumer)
+		if (null != threadNumberConsumer) {
 			noOfConsumerThreads =  (Integer) threadNumberConsumer;
-		// no of threads - Producer
+		}
+		
+		//  no of threads - Producer
 		Object threadNumberProducer = options.valueOf("ntProducer"); 
-		if (null != threadNumberProducer)
+		if (null != threadNumberProducer) {
 			noOfProducerThreads =  (Integer) threadNumberProducer;
-		
-		// maxRecords
+		}
+		 //  maxRecords
```

Comment:
same as above. This looks like the result of a global find/replace? But adding space before all the inline comments makes many of them not align with the code below

Reply:
done

Reply:
done